migrations/sql/V2022.02.10.16.20__add_preambule_text_column_to_permit_amendment_permit.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE PERMIT_AMENDMENT 
+    ADD COLUMN PREAMBLE_tEXT character varying(2000) DEFAULT '' NOT NULL;
```

Comment:
We should confirm the limit with the business and if it's necessary, 2000 seems a little low.

Reply:
I'll confirm the limit. Currently, in the word template we had approx. 1000 chars.